testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -230,14 +230,14 @@ def test_datawriter_universe(filename, tmpdir):
     """
     fn = str(tmpdir.join(filename))
 
-    u = mda.Universe(LAMMPSdata_mini)
+    u = mda.Universe(LAMMPSdata_mini, convert_units = False) # By defaul convert_units = True
```

Comment:
Parametrize over `convert_units = False` and `convert_units = False` with `@pytest.mark.parametrize`